quasar.config.js

```diff
@@ -132,7 +138,7 @@ module.exports = configure(function (/* ctx */) {
       // directives: [],
 
       // Quasar plugins
-      plugins: []
+      plugins: ['Notify'],
```

Comment:
This is your only change here right?.. the rest are spaces

Reply:
yes sir